app/actions/product-actions.js

```diff
@@ -31,8 +31,8 @@ function getItemsCollection(product, options) {
   var items = product.getItemsByStatus(options.status);
 
   if (items.config.get('order_by')) {
-    // Set "Recent" as the default sort
-    let sort = STATUS_MAPPINGS[options.sortField] || 'recent';
+    let previousSortField = window.localStorage.getItem(`itemColumn-${options.status}-sortField`);
```

Comment:
I thought about that, too. Here's why I didn't put it there:

1. We need to know this _before_ we send the call so that we get the right items back from the API.
2. Consider localstorage as if it were another "api" — and this lookup merely being another point of synchronization with an 3rd party data store. Keeping in line with the "actions do server-y things" approach, it's not out of place to see this here. Also worth noting that the call to save this value to localStorage happens later on in this file, so this moves all of the localStorage interactions into one place.